11-node-frameworks/classworks/Gloria_Loi/index.js

```diff
@@ -0,0 +1,49 @@
+const Koa = require('koa');
+const serve = require('koa-static');
+const Router = require('koa-router');
+const session = require('koa-generic-session');
+const bodyParser = require('koa-bodyparser');
+const Joi = require('@hapi/joi');
+
+const router = Router();
+
+const schema = Joi.object({
+  firstname: Joi.string().min(3).required(),
+  lastname: Joi.string().min(3).required(),
+  text: Joi.string().min(3).required(),
+  mark: Joi.number(),
+});
+
+const users = [];
```

Comment:
Please, check the cons of global variables
https://stackoverflow.com/questions/484635/are-global-variables-bad